testpmd/hooks/install.yml

```diff
@@ -11,8 +11,8 @@
 
 - name: Deploy NFV Example CNF catalog
   include_role:
-    name: "{{ dci_config_dir }}/hooks/nfv-example-cnf-deploy/roles/example-cnf-catalog"
+    name: example-cnf-catalog
```

Comment:
What is the benefit of this?  I'm curious because I'm specifying the full path to the CVP roles that I'm using.

Reply:
It improves the readabity of the logs, currenlty this is the output in in the ansible.log file
```
TASK [/var/lib/dci/example-cnf-config/testpmd/hooks/nfv-example-cnf-deploy/roles/example-cnf-validate : wait for migration to com
plete and packet recovery] ***
```

where the below format is more readable
```
TASK [example-cnf-validate : wait for migration to complete and packet recovery] *************************************************
```